readme.txt

```diff
@@ -3,31 +3,38 @@ Contributors: batmoo, joen, automattic, potatomaster, albertomedina, google, xwp
 Tags: amp, mobile
 Requires at least: 4.7
 Tested up to: 5.0
-Stable tag: 0.7.2
+Stable tag: 1.0.0
```

Comment:
We can update this once we actually do the release as part of the release checklist.